setup.cfg

```diff
@@ -53,6 +53,7 @@ install_requires =
     wiimatch>=0.2.1
     packaging>19.0
     importlib-metadata>=4.11.4
+    jsonschema>=4.8
```

Comment:
The lower pin was picked to match what is currently listed in the asdf dependencies:
https://github.com/asdf-format/asdf/blob/b14591c13e3901742426a7c7fcb6aac8754b31b2/pyproject.toml#L35